rules.html

```diff
@@ -3,36 +3,32 @@
 
 <head>
     <title>Rules</title>
+    <link href="styles.css" rel="stylesheet" type="text/css" />
 </head>
 
 <body>
     <div id="rules">
         <h4>Rule and Information:</h4>
```

Comment:
We need to add the rule about stacking special cards. Like if you draw 4, the bot can play a draw 4, then you have to have a draw 4 or risk having to draw 8 cards (because it stacks).